src/pow_fpga_accel.c

```diff
@@ -123,11 +146,37 @@ static bool PoWFPGAAccel_Context_Initialize(ImplContext *impl_ctx)
         goto fail_to_open_odata;
     }
 
+    ctx->pow_info = (PoW_Info *) malloc(sizeof(PoW_Info));
```

Comment:
If the fpga implementation is default for one thread, the `PoW_Info` member can be declared as object rather than pointer and we can avoid this malloc.

Reply:
If so, the implementation of `PoWFPGAAccel_getPoWInfo` is also needed to modify.